howtos/configuration.md

```diff
@@ -10,9 +10,26 @@ The documentation can be customized by setting parameters in [docusaurus.config.
 
 ## Search by Algolia
 
-- API-Key and Index name is configured in `./docusaurus.config.js`.
+Internal search is handled by an integration with Algolia DocSearch.
```

Comment:
Is it worth linking to [Algolia DocSearch](https://docsearch.algolia.com/) here?

Reply:
```suggestion
Internal search is handled by an integration with [Algolia DocSearch](https://docsearch.algolia.com/).
```